oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -92,6 +92,8 @@ public class ComputeEngineCredentials extends GoogleCredentials
   static final Duration COMPUTE_REFRESH_MARGIN = Duration.ofMinutes(3).plusSeconds(45);
 
   private static final Logger LOGGER = Logger.getLogger(ComputeEngineCredentials.class.getName());
+  private static final org.slf4j.Logger SLF4JLOGGER =
+      LoggingConfigs.getLogger(ComputeEngineCredentials.class);
```

Comment:
qq, what's the reason why we can't replace LOGGER with the SLF4JLOGGER?

Reply:
Gotcha, do you know if there is any impact of having these two Loggers working together in the same class? Would they have two different logging formats show up in someone is using ComputeEngineCredentials?

Reply:
They will each show up according to the correponding logging config (formatting, sink ...). They do not interfere with each other.
That said, this is a temporary stage , I will migrate the JUL logging in next release.

Reply:
Ok sounds good to me then.